acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
except right now this cannot be run more than once as the cert creation barfs on existing files. We should wipe it, use a tmp dir, or skip cert generation.